### PR TITLE
Issue card improvements

### LINKED ISSUE #30682: Project board column abel visual bug
### Description

image

### Gitea Version

trunk

### Can you reproduce the bug on the Gitea demo site?

Yes

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

docker

### Database

None

### PR DESCRIPTION
**Pull request by silverwind**
Fixes https://github.com/go-gitea/gitea/issues/30682 and does a few improvements:

- Use gap instead of margin/padding
- Don't render empty image div
- Remove `right floated` class that did nothing

Screenshot 2024-04-24 at 20 21 20

### CODE REVIEW

File: templates/repo/issue/card.tmpl [hunk 1]
```diff
@@ -1,13 +1,16 @@
 {{with .Issue}}
 	{{if eq $.Page.Project.CardType 1}}{{/* Images and Text*/}}
+		{{$attachmentIDs := index $.Page.issuesAttachmentMap .ID}}
+		{{if $attachmentIDs}}
```
Comment (KN4CK3R) [inline templates/repo/issue/card.tmpl]:
```suggestion
		{{$attachments := index $.Page.issuesAttachmentMap .ID}}
		{{if $attachments}}
```
It's list of attachments, not just a list of ids.

File: templates/repo/issue/card.tmpl [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 		<div class="card-attachment-images">
-			{{range (index $.Page.issuesAttachmentMap .ID)}}
+			{{range $attachmentIDs}}
```
Comment (KN4CK3R) [inline templates/repo/issue/card.tmpl]:
```suggestion
			{{range $attachments}}
```

**Review by lunny — APPROVED** [review]

**Review by wxiaoguang — APPROVED** [review]
I do not think these `tw-` tricks are good enough, meanwhile I can accept it as a temporary fix.

**Comment by silverwind** [comment]
The only "trick" here is `tw-gap-[5px]`, but I felt the need to have such a specific value there.

**Pull request merged**